hibernate-core/src/main/java/org/hibernate/EmptyInterceptor.java

```diff
@@ -148,5 +148,9 @@ public void onCollectionRecreate(Object collection, Serializable key) throws Cal
 	@Override
 	public void onCollectionUpdate(Object collection, Serializable key) throws CallbackException {
 	}
-	
+
+	@Override
+	public String inspect(String sql) {
```

Comment:
I'd prefer that we also deprecate `Interceptor#onPrepareStatement`, and invert this by having `EmptyInterceptor#onPrepareStatement` call `EmptyInterceptor#inspect`.  Makes it easier to remove `Interceptor#onPrepareStatement` later; less work.